packages/volto-slate/src/editor/less/slate.less

```diff
@@ -11,13 +11,14 @@ h4 {
   }
 
   a.anchor {
+    display: contents;
     margin-left: -1ch;
     float: left;
 
     svg {
-      width: 1ch;
-      visibility: hidden;
+      width: 1.2ch;
       transform: rotate(45deg);
+      visibility: hidden;
     }
   }
 }
```

Comment:
I would prefer a link icon aligned and with same size of the headline.

Cosmetics: the link icon is rotated. Even if the shape of the icon is unusual, it's a global design decission.

Reply:
@ksuess it's relatively difficult to make the link icon properly wrap with the text, so if it's multiple lines, it doesn't behave well. But my css knowledge may be limited, maybe there's a hack to do it